geomesa-gt/geomesa-gt-tools/src/main/scala/org/locationtech/geomesa/geotools/tools/GeoToolsDataStoreCommand.scala

```diff
@@ -64,7 +64,7 @@ object GeoToolsDataStoreCommand {
       description = "Parameter for DataStoreFinder to load the data store, in the form key=value",
       variableArity = true,
       converter = classOf[KeyValueConverter])
-    val params: java.util.List[(String, String)] = new java.util.ArrayList()
+    var params: java.util.List[(String, String)] = new java.util.ArrayList()
```

Comment:
*blinks*  Why did this change to be a var?

Reply:
the jcommander update didn't like having it be a val...

Reply:
Oooookkkk... Is JCommander mutating state?

Reply:
yeah, pretty much all our param classes use vars. i think b/c this was a list it was ok before since the list was mutable? but the way jcommander works is you pass in your param class and it fills in all the vars based on the command line args.

Reply:
it's mutated once up front, and then theoretically not touched again although there's nothing preventing you i guess...